content/atomics_intro.tex

```diff
@@ -29,19 +29,16 @@
 \ac{AMO} routines each support the ``standard \ac{AMO} types’’ listed in Table \ref{stdamotypes}, 
 except for \FUNC{shmem\_fetch}, \FUNC{shmem\_set}, and \FUNC{shmem\_swap}, which supports the ``extended \ac{AMO} types’’ listed 
 in Table \ref{extamotypes}.
+\newtext{Implementations may optionally support additional types.}
```

Comment:
This is in conflict with Sec. 5, paragraph 1: ```The OpenSHMEM header files for C and Fortran must contain only the interfaces and constant names defined in this specification.```

But, also, I don't think this statement adds anything of substance to the specification.